frontend/src/utils/textEditorUtil.ts

```diff
@@ -0,0 +1,13 @@
+export default class TextEditorUtil {
+  public static getEditorContentIfPresent(
+    value: string | undefined,
+  ): string | undefined {
+    if (value == undefined || value.trim().length == 0) return undefined
+    const divElem = document.createElement("div")
+    divElem.innerHTML = value
+    const hasImgElem = divElem.getElementsByTagName("img").length > 0
+    const hasTable = divElem.getElementsByTagName("table").length > 0
+    const hasInnerText = divElem.innerText.trim().length > 0
+    return hasInnerText || hasImgElem || hasTable ? value : undefined
+  }
+}
```

Comment:
Do we really need this?

Reply:
I know, it is a relict from the old DocumentationUnitCategories update logic, but really the more I think about it the less it makes sense. 

We just want to check if there is content. If yes, it will be either img, table or at least text. So the rest of the code is redundant. The only thing that we want to check here, is there a string and is the string not empty?

`value && value.trim().length > 0 ? value : undefined`

And with that we can remove this whole class.